src/main/java/com/google/api/codegen/discovery/config/ApiaryConfigToSampleConfigConverter.java

```diff
@@ -243,7 +247,10 @@ private MessageTypeInfo createMessageTypeInfo(
     Map<String, FieldInfo> fields = new HashMap<>();
     if (deep) {
       for (Field field2 : type.getFieldsList()) {
-        fields.put(field2.getName(), createFieldInfo(field2, type, method));
+        // TODO(saicheems): We ignore messages as an easy way around cycles for the moment.
```

Comment:
(this method has supported deep exploration, but no function ever actually set `deep=true` before because of this limitation)